src/utils/useStickyTabs.ts

```diff
@@ -33,55 +31,48 @@ const writeTabToSession = (viewIdentifier: string, tab: string): void => {
   }
 };
 
-const useStickyTabs = ({
-  defaultTab,
-  tabs,
-  viewIdentifier,
-  keepQuery = true,
-  tabsAreNewPages = true,
-}: StickyTabsProps) => {
+const useStickyTabs = ({ defaultTab, tabs, viewIdentifier }: StickyTabsProps) => {
   const location = useStateLocation();
   const navigate = useSyncNavigate();
   const query = useQuery();
-  const tab = query.get('tab');
+  const queryTab = query.get('tab');
 
   const [activeTab, setActiveTab] = useState<string>(defaultTab);
 
-  const onTabChange = useCallback(
+  const onChangeTab = useCallback(
     (newTab: string) => {
-      query.set('tab', newTab);
-      const emptyQuery = tab === newTab ? query.toString() : new URLSearchParams(`tab=${newTab}`);
-      navigate(getLocation(location.pathname, location, keepQuery ? query.toString() : emptyQuery.toString()), {
-        replace: !tabsAreNewPages,
-      });
+      if (queryTab) {
+        navigate(getLocation(location.pathname, location), { replace: true });
+      }
```

Comment:
Does this mean that the current tab is only stored in either sessionStorage or in the url, but not both?
I suppose that this also means that every page with tabs now supports navigation to a specific tab just using the url, right?

Reply:
That's great!